.travis.yml

```diff
@@ -15,8 +15,10 @@ install:
   - docker-compose exec web bash -c "rake db:setup"
   - docker-compose exec web bash -c "rake db:migrate"
   - docker-compose exec web bower install --allow-root
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger; fi'
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger danger-junit; fi'
 
 script:
-  - docker-compose run -e CI=TRUE web bash -c "rake test:all"
+  - docker-compose run -e CI=TRUE web bash -c "GENERATE_REPORT=true rake test:all"
+  - echo -e '<?xml version="1.0" encoding="UTF-8"?>' > output.xml
+  - tail -n +2 -q ./test/reports/TEST*.xml >> output.xml
```

Comment:
@jywarren What this does is concatenates all the xml reports into one file that is used by junit to parse the result. ci_reporter generates xml reports for each test file.